src/test/java/com/plaid/client/integration/AbstractItemIntegrationTest.java

```diff
@@ -30,6 +30,7 @@
   protected static void assertAccount(Account actualAccount,
                                       String expectedType,
                                       String expectedSubType,
+                                      Account.VerificationStatus expectedVerificationStatus,
```

Comment:
does this not make other things fail to compile?

Reply:
yes it did 😅 I've updated the PR and made sure it compiles on my own machine... we should get CI in though